app/src/main/java/org/fossasia/openevent/general/auth/AuthFragment.kt

```diff
@@ -50,6 +50,12 @@ class AuthFragment : Fragment(), ComplexBackPressFragment {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [LongMethod - 61/60 - [onCreateView] at /src/app/src/main/java/org/fossasia/openevent/general/auth/AuthFragment.kt:55:5](https://app.codacy.com/app/fossasia/open-event-android-general/pullRequest?prid=4650690)

Reply:
onCreateView is the pre-defined functions with a fixed no of parameters. Codacy is giving error "Methods should have one responsibility. Long methods can indicate that a method handles too many cases at once."